src/yang/ietf-bgp.yang

```diff
@@ -295,9 +295,7 @@ module ietf-bgp {
           }
 
           leaf local-port {
-            type inet:port-number {
-              range "0..65535";
-            }
+            type inet:port-number
```

Comment:
Any particular reason to drop the range check. I know it does not mean much to have a range that spans any possible value.

You are missing a semicolon at the end of the statement. Do you have pyang to validate the models that you are touching?

Reply:
I sloppily failed to run pyang to validate my changes, but will do so after updating the pull request.

The range check is part of the IETF type, and makes zero sense to have since it's not refining within the prior range:

From RFC 6021
   typedef port-number {
     type uint16 {
       range "0..65535";
     }